File: fix-orthanc.js
const fs = require('fs');
const file = '/home/thang/Minipacs-v.2/config/orthanc.json';
const config = JSON.parse(fs.readFileSync(file, 'utf8'));
config.AuthenticationEnabled = false;
if (!config.HttpExtraHeaders) config.HttpExtraHeaders = {};
config.HttpExtraHeaders['Access-Control-Allow-Origin'] = '*';
fs.writeFileSync(file, JSON.stringify(config, null, 2));

File: fix-orthanc.py
import json

file = '/home/thang/Minipacs-v.2/config/orthanc.json'
with open(file, 'r') as f:
    config = json.load(f)

config['AuthenticationEnabled'] = False
if 'HttpExtraHeaders' not in config:
    config['HttpExtraHeaders'] = {}
config['HttpExtraHeaders']['Access-Control-Allow-Origin'] = '*'

with open(file, 'w') as f:
    json.dump(config, f, indent=2)
